src/tools/error_index_generator/book_config.toml

```diff
@@ -0,0 +1,19 @@
+[book]
+title = "Error codes index"
+description = "Book listing all Rust error codes"
+src = ""
+
+[output.html]
+git-repository-url = "https://github.com/rust-lang/rust/"
+additional-css = ["extra.css"]
+additional-js = ["extra-info.js"]
```

Comment:
> Can we give these two files a more descriptive name than `extra`? For example, name them `error-index.css` and `error-index.js`?

Very good idea!

> Alternatively, can we make mdBook put all the error-index related assets in a subdirectory? For example, we could make `error_codes` be the book output directory, including putting the `error-index.html` in there, and just leave the old top-level `error-index.html` as a redirect to point inside the book?

That would break all the current links no? But otherwise, it can be done indeed.

Reply:
Ok!